lib/cluster/vpc.rb

```diff
@@ -108,12 +122,16 @@ def self.get_parameters
       }
     end
 
-    def self.get_template_path
+    def self.get_cf_template
       if supports_efs?
-        './templates/OpsWorksinVPCWithEFS.template'
-      else
-        './templates/OpsWorksinVPC.template'
+        return File.read('./templates/OpsWorksinVPCWithEFS.template.erb')
       end
+      erb = Erubis::Eruby.new(File.read('./templates/OpsWorksinVPC.template.erb'))
```

Comment:
The cloudformation template is now generated from an erb template to allow generating the security groups w/ the correct IPs